components/Extensions/src/Markup/Abstract/TextIconExtension.cs

```diff
@@ -33,12 +34,12 @@ protected static FontFamily SymbolThemeFontFamily
     /// <summary>
     /// Gets or sets the thickness of the icon glyph.
     /// </summary>
-    public FontWeight FontWeight { get; set; } = FontWeights.Normal;
+    public Windows.UI.Text.FontWeight FontWeight { get; set; } = Windows.UI.Text.FontWeights.Normal;
```

Comment:
Should these 2 switch namespaces based on if it is WinAppSDK or not?

Reply:
The ifdef at the top was meant to handle this originally. It wasn't clear why it allowed this to build before preview1, but it was the only thing that worked.

It seems like the `FontWeight` struct doesn't have a `Microsoft.UI.Text` equivalent. Docs for `Microsoft.UI.Text.FontWeights` also link to the `Windows.UI.Text.FontWeight` struct.

Reply:
Right, FontWeight is from Windows.UI.Text but FontWeight**s** is from Microsoft.UI.Text.

```suggestion
    public Windows.UI.Text.FontWeight FontWeight { get; set; } = Microsoft.UI.Text.FontWeights.Normal;
```

Reply:
I see now, must have been a mixup here amid the other errors. Got this pushed, builds with the workaround under experimental2 but not preview1.

Reply:
Are you getting the type conflict error? I see you moved to the stable Windows SDK projection.  Can you keep that at the preview still?  WinAppSDK preview 1 doesn't have the changes yet for that updated Windows SDK projection.  We redirected a couple types from WinAppSDK to the Windows SDK projection so we had to ship the Windows SDK projection first before WinAppSDK can take advantage of them and mark them as forwarded.

i.e can you downgrade to

    <WindowsSdkPackageVersion>10.0.22621.37-preview</WindowsSdkPackageVersion>

Reply:
I see, that helps explain some of the errors I've been seeing. Pushed up.